src/general/StageBase.cs

```diff
@@ -252,9 +252,9 @@ public override void _Process(float delta)
 
         if (debugOverlay.PerformanceMetricsVisible)
         {
-            var entities = rootOfDynamicallySpawned.GetChildrenToProcess<ISpawned>(Constants.SPAWNED_GROUP).Count();
+            var entities = rootOfDynamicallySpawned.GetChildrenToProcess<ISpawned>(Constants.SPAWNED_GROUP).ToList();
```

Comment:
I'd prefer to avoid allocating memory each frame to a new list, but I suppose as this is only used when debugging, this is fine...

Reply:
I had a lot of trouble with the linter with this line and those around it, and this was the only configuration I found that passed. So please let me know if there is an alternative that will also pass.

Reply:
I think that a manual loop that increments an int and adds a weight value to another variable, would be the only way I can think of that avoids allocating extra memory or looping all of the entities twice.